sdk/storage/Azure.Storage.Files.DataLake/src/Models/AccessControlChangeOptions.cs

```diff
@@ -20,5 +20,14 @@ public struct AccessControlChangeOptions
         /// Empty value indicates that maximum number of batches in unbound and operation continues till end.
         /// </summary>
         public int? MaxBatches { get; set; }
+
+        /// <summary>
+        /// Optional. Valid for "SetAccessControlRecursive" operation. If set to false, the operation will terminate quickly
+        /// on encountering user errors (4XX). If true, the operation will ignore user errors and proceed with the operation
+        /// on other sub-entities of the directory. Continuation token will only be returned when forceFlag is true in case of user errors.
+        /// If not the service will set the default value is false for this.
+        /// </summary>
+        public bool? forceFlag { get; set; }
```

Comment:
Few things to tweak:
1. The doc seems to be copied verbatim from swagger. However, we do abstract rest api complexity pretty well in the sdk so statements like "Valid only for SetAccessControlRecursive" are not very accurate and might be confusing - we hide those details for customer. I'd just leave part that describes the behavior and remove the rest. Also I'd use "failure" instead of "error" to describe failures on some paths - these are expected results for some conditions so calling them errors doesn't feel accurate.
2. Naming. Server side named it "forceFlag" however the "flag" is excessive - we know that from bool type. I'd just call it `force` or `continueOnFailure` (if we want to be more descriptive). 
3. I'd just use `bool` instead of `bool?`. Sending false and not sending anything for that parameter does exactly the same thing. So we could just default to false and send the param if it's set to true.

Reply:
I actually like `continueOnFailure` more after looking at tests.

Reply:
btw. are we actually passing this flag in rest call? I don't see any mapping change in this diff - I'd expect it in pathclient.

Reply:
When trying to make a new branch with my changes, I forgot my changes to DataLakePathClient.cs, That's why the value wasn't being passed in the rest call. I'm surprised I was able to pass the tests I wrote. I might have to update my tests tomorrow.